nixos/modules/services/misc/nzbget.nix

```diff
@@ -51,6 +51,8 @@ in
           MainDir = "/data";
         };
       };
+
+      package = mkPackageOption pkgs "nzbget" { };
```

Comment:
Could you move this between `enable` and `user`? It's not really a written rule, but I think it's generally a convention to have the `package` option` right after the `enable` option.

Reply:
Done